lib/endpoints/boot_params.js

```diff
@@ -5,16 +5,70 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright 2020 Joyent, Inc.
  */
+var util = require('util');
 
 var restify = require('restify');
 
 var ModelServer = require('../models/server');
 var common = require('../common');
 var validation = require('../validation/endpoints');
 var ModelPlatform = require('../models/platform');
-var dns = require('dns');
+
+/*
+ * Expected boot modules format is as follows:
+ *
+ * {
+ *      "path": "boot module path to be used by booter",
+ *      "type": "Content type. Right now only base64 is supported",
+ *      "content": "File contents, if necessary encoded as specified by type"
+ * }
+ *
+ * In the future, support for new content types or additionaly members for
+ * the boot_module objects could be provided, being the defaults the originally
```

Comment:
can we rewrite that last part?
Maybe ", defaulting to base64." to be clear?

Reply:
Of course. Done!